src/controller/java/AndroidOperationalCredentialsIssuer.cpp

```diff
@@ -225,9 +225,8 @@ CHIP_ERROR AndroidOperationalCredentialsIssuer::CallbackGenerateNOCChain(const B
     jbyteArray javaAttestationElements;
     JniReferences::GetInstance().N2J_ByteArray(env, attestationElements.data(), attestationElements.size(),
                                                javaAttestationElements);
-    // new log message below
-    ChipLogError(Controller, "AndroidOpCredsIssuer attestationElements size %d/%d", static_cast<int>(attestationElements.size()),
-                 static_cast<int>(env->GetArrayLength(javaAttestationElements)));
+    ChipLogProgress(Controller, "AndroidOpCredsIssuer attestationElements size %d/%d", static_cast<int>(attestationElements.size()),
+                    static_cast<int>(env->GetArrayLength(javaAttestationElements)));
```

Comment:
Please remove this log. It's a "regular success tracing log". Why is this necessary? The output of this cannot be used. If there is a bug, let's fix the bug.

Reply:
removed